besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2783,10 +2783,19 @@ private Optional<PkiBlockCreationConfiguration> maybePkiBlockCreationConfigurati
   }
 
   private SynchronizerConfiguration buildSyncConfig() {
+    Integer fastSyncMinPeers = fastSyncMinPeerCount;
+    if (fastSyncMinPeers == null) {
+      if (isMergedNetwork(network)) {
```

Comment:
Is there more logic required here for the case where the sync begins at a pre-merge block? I've had cases recently on Ropsten where using fast sync with <=2 peers resulted in the sync stalling (ended up on a fork perhaps?)

Reply:
we cannot start a sync on a premerge block because of the snapsync. We need to use <128 old blocks. So after small amount of time it's not possible

Reply:
if the network merged the pivot block need to come from CL so 1 it's enough .